src/main/java/com/minecolonies/coremod/entity/ai/citizen/miner/EntityAIStructureMiner.java

```diff
@@ -975,7 +976,7 @@ private void setBlockFromInventory(@NotNull final BlockPos location, final Block
         {
             getInventory().extractItem(slot, 1, false);
             //Flag 1+2 is needed for updates
```

Comment:
This comment doesn't apply anymore, does it?

Reply:
ah ye no longer needed